.github/workflows/deploy.yaml

```
@@ -36,10 +36,25 @@ jobs:
           fetch-depth: 0
 
       - name: Deploy
-        run: ./scripts/deploy.sh
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:24: Double quote to prevent globbing and word splitting

(shellcheck)